Algorithm.Python/stubs/QuantConnect/Securities/____init___6.py

```diff
@@ -312,7 +312,7 @@ def __init__(self, timeKeeper: QuantConnect.Interfaces.ITimeKeeper) -> QuantConn
     Item: indexer#
 
 
-class SecurityPortfolioManager(QuantConnect.ExtendedDictionary[SecurityHolding], System.Collections.IEnumerable, QuantConnect.Interfaces.IExtendedDictionary[Symbol, SecurityHolding], System.Collections.Generic.ICollection[KeyValuePair[Symbol, SecurityHolding]], System.Collections.Generic.IDictionary[Symbol, SecurityHolding], QuantConnect.Securities.ISecurityProvider, System.Collections.Generic.IEnumerable[KeyValuePair[Symbol, SecurityHolding]]):
+class SecurityPortfolioManager(QuantConnect.ExtendedDictionary[SecurityHolding], System.Collections.IEnumerable, QuantConnect.Interfaces.IExtendedDictionary[Symbol, SecurityHolding], System.Collections.Generic.ICollection[KeyValuePair[Symbol, SecurityHolding]], System.Collections.Generic.IDictionary[Symbol, SecurityHolding], System.Collections.Generic.IEnumerable[KeyValuePair[Symbol, SecurityHolding]], QuantConnect.Securities.ISecurityProvider):
```

Comment:
Bug in python stubs generator. We should sort the class inheritance structure to avoid this issue.